BUILD_DETAIL.md

```diff
@@ -68,8 +68,9 @@ $ bin/cucumber
 
 ## YARD documentation
 
-RSpec uses [YARD](http://yardoc.org/) for API documentation on the [rspec.info site](http://rspec.info/).
-Our commitment to [SemVer](htp://semver.org) requires that we explicitly
+RSpec uses [YARD](https://yardoc.org/) for API documentation on the [rspec.info
+site](https://rspec.info/).
+Our commitment to [SemVer](https://semver.org) requires that we explicitly
```

Comment:
Reformatted so it's 80 char width like the rest of the document.

Reply:
Preferred as was.